lib/components/artist/about.js

```diff
@@ -36,26 +36,12 @@ class About extends React.Component {
   }
 
   auctionResults() {
-    // Bail early
-    if (this.props.artist.is_display_auction_link === false) { return null }
-    // TODO: Refactor into:
-    //<NavButton title="Auction Results" href={`/artist/${this.props.artist.id}/auctions`} />
-    return (
-      <TouchableWithoutFeedback onPress={this.openAuctionsURL}>
-        <View style={{ marginBottom: 20, marginLeft: 20, marginRight: 20 }}>
-          <Separator />
-          <View style={{ flexDirection: "row", justifyContent: "space-between", padding: 0 }}>
-            <Text style={{ marginLeft:-20, fontFamily: 'Avant Garde Gothic ITCW01Dm',  marginTop: 14, marginBottom:14 }} >AUCTION RESULTS</Text>
-            <Image style={{alignSelf: 'center', marginRight:-20}} source={require('../../../images/horizontal_chevron.png')} />
-          </View>
-          <Separator />
-        </View>
-      </TouchableWithoutFeedback>
-    )
-  }
-
-  openAuctionsURL = () => {
-    SwitchBoard.presentNavigationViewController(this, `/artist/${this.props.artist.id}/auction-results`)
+    if (this.props.artist.is_display_auction_link) {
```

Comment:
What does `is_display_auction_link` mean in this context? Does it really indicate that we should show auction results for an artist or is it only tangentially related?